oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/FormValidation.js

```diff
@@ -0,0 +1,35 @@
+import * as Yup from "yup";
+import { checkDuplicateLanguage } from "../utils";
+import { i18next } from "@translations/oarepo_vocabularies_ui/i18next";
+
+export const MyFormSchema = Yup.object().shape({
+  title: Yup.array()
+    .of(
+      Yup.object().shape({
+        language: Yup.string().required(i18next.t("required")),
+        title: Yup.string().required(i18next.t("required")),
+      })
+    )
+    .test(
+      "same language",
+      (value) => {
+        return [
+          value.value.map((item) => ({
+            language: i18next.t("languageError"),
+          })),
+        ];
+      },
+      (value, context) => {
+        return checkDuplicateLanguage(value);
+      }
+    ),
+  props: Yup.object().shape({
+    ICO: Yup.string()
+      .length(8, ({ length }) => i18next.t("lengthError", { length: length }))
+      .matches(/^\d+$/, i18next.t("numbersError")),
```

Comment:
I thought it would be useful to have one error message for the length of string i.e. same message for ICO and RID and I would only interpolate the actual desired length value. Do I understand well that we want to have separate messages for length of ICO and RID field. Also, I am not sure how verbose are we supposed to make the translation keys i.e. is it OK for key to be named containsOnlyNumbersError? How would you call this key please just to get a sense?

Reply:
@Ducica I get your point, in that case, I would name it like `invalidLengthError` and `notANumberError`, with possible translations of `must be {length} characters long` and `value must be a number`